ta_list_methods.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from collections import defaultdict
+import act
+
+
+# creates an api agent for use in searches
```

Comment:
I propose to change all method comments to [docstrings](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring). In this way they are also available when using help(method) from an interpreter.